### DISCUSSION TITLE
Identify METplus version

### DISCUSSION THREAD

**Discussion by jwraby** [dtcenter/METplus#2104] [Training] [answered]
To prepare for the upcoming METPLUS ADVANCED TRAINING SERIES, I tried to find the METplus version I have running as a container on the DoD HPC, but I can't find it. I pulled the container from the dtcenter on 13 MAR 2023. The MET version on it is V11.0.1. How can I find the METplus version? Thanks.

**Comment by georgemccabe** [comment, marked as answer]
Hi @jwraby,

The version of METplus is output in one of the first log entries for a given run.

> 04/05 17:19:57.882Z metplus INFO: Running METplus **v5.0.1** as user mccabe(15832) with command: /home/mccabe/METplus/ush/run_metplus.py ...

You can also check the version of METplus by running the `run_metplus.py` script without any arguments:

*run_metplus.py*

> Running METplus **5.0.1**
> 
> Usage: run_metplus.py arg1 arg2 arg3
>     -h|--help               Display this usage statement
> 
> Arguments:
> /path/to/parmfile.conf -- Specify custom configuration file to use
> section.option=value -- override conf options on the command line
> 

The version is also stored in the metplus/VERSION text file in the installation location. In the container, it should be `/metplus/METplus/metplus/VERSION`.

**Reply by jwraby** [reply]
Thanks for this helpful info, George. I shelled into the container and went to  /metplus/METplus/ush. I ran the python script you suggested and it output the usage dialog which had the version "METplus 5.0.1".
